docs/_docs/step-by-step/02-liquid.md

```diff
@@ -66,6 +66,18 @@ Now it's your turn, change the Hello World! on your page to output as lowercase:
 ```  
 {% endraw %}
 
+To get our changes processed by Jekyll we need to add [front matter](03-front-matter/) to the top of the page:
+
+{% raw %}
```

Comment:
The `raw` tags are only necessary if you have *Liquid constructs* that need to be *escaped*. The following is sufficient:

    ```markdown
    ---
    # Front Matter tells Jekyll to process Liquid
    # and allow converting Markdown into HTML
    ---
    ```